test/extended/openstack/egressip.go

```diff
@@ -73,6 +73,12 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack][egressip] An egre
 
 	g.It("attached to a floating IP should be kept after EgressIP node failover with OVN-Kubernetes NetworkType", func() {
 
+		dualstackIpv6Primary, err := isIpv6primaryDualStackCluster(ctx, oc)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		if dualstackIpv6Primary { //This test is covering and scenario that has no sense with ipv6 as there is no FIP/VIP association.
```

Comment:
typo: a scenario

I'm not sure I follow why this test wouldn't be applicable for IPv6. Where there is no FIP/VIP?

Reply:
This testcase is checking that when an egressIP moves to other worker, Openshift is not changing the neutron port so the connection between the egressIP and the FIP is not broken.

On IPV6, we don't have FIP so there is no need to test it in ipv6primary.

I notice that "VIP" is not precise, so I am changing the comment.